src/pcapi/scripts/temporary_script.py

```diff
@@ -0,0 +1,44 @@
+from sqlalchemy.orm import load_only
+
+from pcapi.core.bookings.models import Booking
+from pcapi.core.bookings.models import IndividualBooking
+from pcapi.models import db
+
+
+def get_base_query(min_id):
+    return Booking.query.filter(
+        Booking.userId.isnot(None),
+        Booking.individualBookingId.is_(None),
+        Booking.id > min_id,
+    )
+
+
+def get_ids_query(min_id):
+    return get_base_query(min_id).order_by(Booking.id).with_entities(Booking.id)
+
+
+def migrate_bookings_data(batch_size=1000):
+    item_count = get_ids_query(0).count()
+    print(f"{item_count} bookings to update")
+    if item_count == 0:
+        return
+    modified_sum = 0
+    min_id = 0
+    item_ids = get_ids_query(min_id).limit(batch_size).all()
+    max_id = item_ids[-1][0]
+    while len(item_ids) > 0:
+        bookings: list[Booking] = (
+            get_base_query(min_id).filter(Booking.id <= max_id).options(load_only(Booking.userId, Booking.id)).all()
+        )
+        modified_count = len(bookings)
+        for booking in bookings:
+            individual_booking = IndividualBooking(userId=booking.userId)
+            booking.individualBooking = individual_booking
+            db.session.add(individual_booking, booking)
```

Comment:
```suggestion
            db.session.add(booking)
```

Ce n'est pas important, mais normalement vu qu'il y a un lien entre les deux l'enregistrement est cascadé

Reply:
Ah yes, je modifie en local du coup